### PR TITLE
Tidy dependencies 

### PR DESCRIPTION
**Pull request by IljaN**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,16 +6,17 @@ require (
 	contrib.go.opencensus.io/exporter/jaeger v0.2.0
 	contrib.go.opencensus.io/exporter/ocagent v0.6.0
 	contrib.go.opencensus.io/exporter/zipkin v0.1.1
-	github.com/UnnoTed/fileb0x v1.1.4 // indirect
```
Comment (C0rby) [inline go.mod:9]: I think fileb0x should stay in the go.mod file.
It is used here: https://github.com/owncloud/ocis-konnectd/blob/105edd6195b7da588ec716c8a2c9a17fd4c9d062/pkg/assets/assets.go

Comment (refs) [reply go.mod:9]: 👍 yup, create a `tools.go` file and add a blank import for it @IljaN just like we do [here](https://github.com/owncloud/ocis-accounts/blob/master/tools.go)

Comment (IljaN) [reply go.mod:9]: Thanks! Done.

**Review by C0rby — APPROVED** [review]
:+1:

**Review by refs — APPROVED** [review]

**Pull request merged**